### DISCUSSION TITLE
When will support for Laravel 11 be available?

### DISCUSSION THREAD

**Discussion by damianidczak** [inertiajs/inertia#1833] [Help]
When will support for Laravel 11 be available?

**Comment by RobertBoes** [comment]
It already is available: https://github.com/inertiajs/inertia-laravel/releases/tag/v1.0.0
Also, this is the wrong repo for that.

**Reply by damianidczak** [reply]
thanks! Right!

**Reply by aadrian** [reply]
> Also, this is the wrong repo for that.

Only this "main" repo has Discussions active :)
